app/main/views/services.py

```diff
@@ -189,12 +189,7 @@ def get_service(service_id):
         Service.service_id == service_id
     ).first_or_404()
 
-    if service.framework.status == 'live':
-        return jsonify(services=service.serialize())
-    elif service.framework.status == 'expired':
-        abort(410)
-    else:
-        abort(404)
+    return jsonify(services=service.serialize())
```

Comment:
Frontend apps will need to change to check statuses - are you on that already? (I'm particularly thinking of buyer app service pages, but supplier app might also use this endpoint in the turned-off edit live services flow.)